modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -91,40 +144,51 @@ endif::[]
 
 
 ifeval::[{suma-content} == true]
-
+.Procedure: Alternative Method for Adding {ubuntu} Channels and Repositories (Unsupported)
+.  
```

Comment:
This is a hack until we find why how to avoid the title procedure getting inside the warning.

Reply:
We should avoid admonitions in procedures. Can't we just put it above the procedure title?

Reply:
https://github.com/SUSE/spacewalk/issues/10522